apps/frontend/src/components/cards/controltable/ControlRowHeader.vue

```diff
@@ -268,5 +268,17 @@ export default class ControlRowHeader extends mixins(HtmlSanitizeMixin) {
       return {label: cci, url: '', description: this.descriptionForTag(cci)};
     });
   }
+  showLegacy(control: any) {
+    let legacyTag = control.data.tags['legacy'];
+    if (legacyTag) {
+      if (Array.isArray(legacyTag)) {
+        return '(' + legacyTag.find((ele) => ele.indexOf('V-') == 0) + ')';
```

Comment:
Is it guaranteed we can find legacy ID that begins with "V-" if the array is present?